.github/workflows/deploy-lambda-assertion.yml

```diff
@@ -0,0 +1,68 @@
+name: Deploy Lambda assertion
+
+on:
+    push:
+      branches:
+        - "main"
+      paths:
+        - "**/src/oneid/oneid-lambda-assertion/**"
+        - "**/src/oneid/oneid-common/**"
+    workflow_dispatch:
+      inputs:
+        environment:
+          description: 'Choose environment'
+          type: choice
+          required: true
+          default: dev
+          options:
+          - dev
+jobs:
+    build:
+      runs-on: ubuntu-22.04
+
+      steps:
+      - name: Checkout code
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332
+
+      - name: Set up Python
```

Comment:
I think this step is not required since we are not going to run the python code nor to install any packages.